wagtail_newsletter/campaign_backends/__init__.py

```diff
@@ -28,7 +28,7 @@ class AudienceNotFoundError(LookupError):
     pass
 
 
-class CampaignBackend:
+class CampaignBackend(metaclass=ABCMeta):
```

Comment:
Just out of curiosity: what's the reason for using the slightly more complex `metaclass=ABCMeta` way, rather than simply extending `ABC` ? :slightly_smiling_face:

Reply:
Because I didn't know any better 😅 Changed to the simpler option, thanks!